src/browser/modules/Carousel/styled.tsx

```diff
@@ -437,6 +437,9 @@ export const StyledSidebarSlide = styled.div.attrs({
     list-style-position: inside;
     padding-left: 0;
     margin-top: 0.5em;
+    p {
+      display: inline-block;
```

Comment:
Before we added list-style-position: inside; <p> in a <li> was shown inline, added this to keep them inline.